README.md

```diff
@@ -431,6 +462,10 @@ $ vi environments/kitchen.json
         "ip": "33.33.33.16",
         "username": "admin",
         "password": "salim"
+      },
+      "supermarket": {
+        "fqdn": "33.33.33.17",
+        "ip": "33.33.33.17"
```

Comment:
Nothing to change in this pr. But we need to make fqdn honor ip or host if it is set. There's an issue filed from suncorp re this for delivery and chef-server: Issue #94